docs/api.rst

```diff
@@ -10,6 +10,7 @@ New Routines
 
 .. autofunction:: adjacent
 .. autofunction:: always_iterable
+.. autofunction:: arithmetic_sequence(start, stop, step)
```

Comment:
If someone knows how to convince Sphinx to do a second signature without duplicating the whole function, please let me know; I couldn't figure it out.